packages/wonder-blocks-modal/util/constants.js

```diff
@@ -0,0 +1,8 @@
+// @flow
+
+/**
+ * The attribute used to identify a modal launcher portal.
+ */
+const ModalLauncherPortalAttributeName = "data-modal-launcher-portal";
+
+export {ModalLauncherPortalAttributeName};
```

Comment:
This is now the only place that the attribute name is specified. So we won't have weird typo errors and such if we ever change this, and we now have something that links all these places to make it clear they are talking about the same thing.